rest/client/testdata/logging.yaml

```diff
@@ -4,7 +4,9 @@ loggers:
   agent:
     - type: evergreen
   system:
-    - type: file
+    - type: splunk
+      splunk_server: www.example.com
+      splunk_token: ${foo}
```

Comment:
Do you test that this is populated in a test?

Reply:
I previously logged to verify that it was populated since the interfaces didn't make this testable, but I refactored a bit to add an assertion